grails-app/controllers/org/broadinstitute/orsp/api/ClarificationController.groovy

```diff
@@ -60,4 +60,28 @@ class ClarificationController extends AuthenticatedController {
         }
         response
     }
+
+    def collectionRequestClarification() {
+        Issue issue = queryService.findByKey(params.id)
+        if (params.comment) {
+            Comment comment = persistenceService.saveComment(issue.projectKey,  getUser()?.displayName, params.comment)
+            String toAddresses = userService.findUser(params.pm)?.collect {it.emailAddress}
+            String fromAddress = (String) getUser()?.emailAddress
```

Comment:
emailAddress is already a string ... why the cast? Remember, casting is *always* a code smell. If you're doing it, you should be 1000% sure that's the only way to do what you need.